src/storm/storage/SparseMatrix.h

```diff
@@ -632,6 +632,11 @@ class SparseMatrix {
      */
     void makeRowGroupingTrivial();
 
+    /*!
+     * Creates a new square matrix by merging the entries in a rowgroup (adding the entries)
+     */
+    SparseMatrix getSquareMatrix();
```

Comment:
I find the name getSquareMatrix very confusing... :-)

Reply:
Do you have a suggestion for the name?